concepts/05 Widgets/ContextMenu/10 Open and Close the Context Menu/30 Events.md

```diff
@@ -1,27 +1,28 @@
 To execute certain commands before or after the **ContextMenu** was opened/closed, handle the [showing](/api-reference/10%20UI%20Widgets/dxContextMenu/4%20Events/showing.md '/Documentation/ApiReference/UI_Widgets/dxContextMenu/Events/#showing'), [shown](/api-reference/10%20UI%20Widgets/dxContextMenu/4%20Events/shown.md '/Documentation/ApiReference/UI_Widgets/dxContextMenu/Events/#shown'), [hiding](/api-reference/10%20UI%20Widgets/dxContextMenu/4%20Events/hiding.md '/Documentation/ApiReference/UI_Widgets/dxContextMenu/Events/#hiding') or [hidden](/api-reference/10%20UI%20Widgets/dxContextMenu/4%20Events/hidden.md '/Documentation/ApiReference/UI_Widgets/dxContextMenu/Events/#hidden') event. If the event handling function is not going to be changed during the lifetime of the widget, assign it to the corresponding **on*EventName*** option when you configure the widget.
 
 ---
```

Comment:
This snippet accordion is broken. I think it's line [136](https://github.com/DevExpress/devextreme-documentation/pull/441/files#diff-005e670db07f742aec2187f570154359R136)
![image](https://user-images.githubusercontent.com/33040934/75984873-3f00ad80-5efc-11ea-9298-af12ceb1341e.png)